echoregions/__init__.py

```diff
@@ -4,8 +4,6 @@
 that contain the majority of the functionality within echoregions.
 """
 
-from _echoregions_version import version as __version__  # noqa
-
```

Comment:
I am not sure how this address the import problem?

Reply:
I think you're right. This removal doesn't need to be made and I'm placing it back. I'm removing the echoregions version py file and placing this in the gitignore since this is generated during the initial pip install.